### PR TITLE
BETA: Register riddle.is-a.dev

### PR DESCRIPTION
**Pull request by r-ddle**
Added `riddle.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by PinkLittleKitty** [comment]
Netlify is not supported.

**Review by PinkLittleKitty — CHANGES_REQUESTED** [review]
Netlify is not supported

**Comment by r-ddle** [comment]
> Netlify is not supported

oh, what are the hostings that you support?

**Comment by PinkLittleKitty** [comment]
I recommend using github pages, but you can also use Railway or Cloudfare.

File: domains/riddle.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "r-ddle",
+        "email": "an7r1ddl3@gmail.com"
+    },
+    "record": {
+        "CNAME": "riddle cname r-ddle.netlify.app."
```
Comment (21Z) [inline domains/riddle.json:7]: 1. Invalid `CNAME` record
2. Netlify is not supported

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by TerraPlayz** [comment]
Inactive

**Pull request closed without merging**